docs/dev.md

```diff
@@ -58,12 +58,14 @@ To start dev mode on a multi module project by using the short-form `liberty` na
 $ mvn liberty:dev
 ```
 
-Liberty server configuration files (such as `server.xml`) will be used from the module that does not have any other modules depending on it.  If there is more than one module without other modules depending on it, specify which module with Liberty configuration that you want to use by including the parameters `-pl <module-with-liberty-config> -am`.  
+Liberty server configuration files (such as `server.xml`) will be used from the module that does not have any other modules depending on it.  If there is more than one module without other modules depending on it, specify which module with Liberty configuration that you want to use by including the parameters `-pl :<module-with-liberty-config-artifact-id> -am`.  
 For example, to use Liberty configuration from a module named `ear`, run:
```

Comment:
Perhaps this should say `module id ear` because `named` seems to suggest file name.